requirements-ci.txt

```diff
@@ -1,4 +1,4 @@
 -r requirements.txt
 -r requirements-tests.txt
-libtorrent==2.0.7
-pytest==7.4.2
+pytest
+libtorrent<2.0.9
```

Comment:
libtorrent is pinned also due to a test failure, but the change was introduced in 2.0.9. We should eventually fix our test rather than keep pinning, but we can still update libtorrent even if just a bit.

Reply:
These are pinned to avoid breaking the CI over time so can be version bumped but must still be pinned

Reply:
> These are pinned to avoid breaking the CI over time so can be version bumped but must still be pinned

For now, I'm happy to change it to `libtorrent==2.0.8`, but I think there's a discussion to be had about this (not in the context of this PR): I can understand the desire for the CI to not break easily, especially if you have little time as-is, but IMO a CI shouldn't easily break (if at all) due to a dependency publishing new bug-fix level releases. 

You're pinned to a two-year old version of what is quite probably your most important dependency. There has to be a better way to approach it (I have a few different ideas, but I don't really know where is the best place to start a discussion such as this?)